Makefile

```diff
@@ -103,6 +104,10 @@ build_docs:
 build_docs_pdf:
 	podman run -v $(CURDIR)/docs:/docs:Z --rm $(DOCS_PDF_CONTAINER) -D /$(DOCS_BUILD_DIR) -o doc.pdf $(DOCS_BUILD_TARGET)
 
+.PHONY: docs_serve
+docs_serve: build_docs
+	podman run -v $(CURDIR)/docs:/docs:Z --rm -p 8088:8088/tcp $(DOCS_SERVE_CONTAINER)
+
```

Comment:
If podman run was successful you can add a `@echo "You can now connect to http://localhost:8088 with your web browser to check the documentation"`

Reply:
So it already prints something like below, when it runs:
```
❯ make docs_serve 
podman run -v /home/anjan/github.com/code-ready/crc/docs:/docs:Z --rm registry.gitlab.com/gbraad/asciidoctor-centos:latest -b html5 -D /docs/build -o index.html /docs/source/getting-started/master.adoc
podman run -v /home/anjan/github.com/code-ready/crc/docs:/docs:Z --rm -p 8088:8088/tcp quay.io/crcont/docs-server:latest
Serving HTTP on 0.0.0.0 port 8088 (http://0.0.0.0:8088/) ...
``` 
Do you still think we need to add that message (localhost, 0.0.0.0, 127.0.0.1 all of these would work)

Reply:
ah, I did not notice there was a message, I agree another one is not needed. If you add a @ before the 'podman' command, I think it will hide the command while still printing the "Serving ..." message.